lib/model/src/main/java/io/atlasmap/v2/Property.java

```diff
@@ -14,6 +14,8 @@
 
     protected String scope;
 
+    protected DataSourceType documentType;
```

Comment:
If you reuse `DataSourceType` then I would suggest to use `dataSourceType` as a property name here. Otherwise, 1) Mark `DataSourceType` deprecated and 2) Create a common enum `DocumentType`

Reply:
changed to dataSourceType